pom.xml

```
@@ -751,6 +771,7 @@
         <spark340.version>3.4.0</spark340.version>
         <spark341.version>3.4.1</spark341.version>
         <spark342.version>3.4.2</spark342.version>
+        <spark343.version>3.4.3-SNAPSHOT</spark343.version>
```

3.4.3 has been released.
```suggestion
        <spark343.version>3.4.3</spark343.version>
```